script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -481,7 +481,7 @@ Now that I know what's got her down, I start to feel the urge to do something ab
 [sha:77e0418fb1c74797061a1990c64e955f96213b68]{
 -- Page 463, Offset 17724.
 -- 俺も今度は意識を集中して視る。もしかしたら何か見つけられるかもしれない」
-I'll see if I can't sharpen my senses this time around as well.
+I'll see if I can sharpen my senses this time around as well. 
```

Comment:
I don't know the linguistic name for it, but "see if we can't X" and "see if we can X" can both mean the same thing in English :upside_down_face: 
I guess if people are liable to think it's wrong then better to be safe.

Reply:
Changed it because it took me a while to get it lol, if it's just me then I think reverting is fine